doc/src/manual/control-flow.md

```diff
@@ -567,17 +574,16 @@ below all interrupt the normal flow of control.
 For example, the [`sqrt()`](@ref) function throws a [`DomainError`](@ref) if applied to a negative
 real value:
 
-```julia
+```jldoctest
 julia> sqrt(-1)
 ERROR: DomainError:
 sqrt will only return a complex result if called with a complex argument. Try sqrt(complex(x)).
- in sqrt(::Int64) at ./math.jl:278
```

Comment:
I find at least the first line or two of the backtrace to be pretty useful, it tells you where you go look at how the code is implemented if you want to look, and it's representative of what happens when you run this code yourself. The line numbers thing we should address in terms of how we run the doctests and how picky we are about everything matching. I think we should detect line numbers in the output and be permissive of them changing, everywhere except on release-branch builds.